src/Microsoft.Health.Fhir.TemplateManagement/Models/DefaultTemplateInfo.cs

```diff
@@ -0,0 +1,38 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using Microsoft.Health.Fhir.Liquid.Converter.Models;
+
+namespace Microsoft.Health.Fhir.TemplateManagement.Models
+{
+    public class DefaultTemplateInfo
+    {
+        /// <summary>
+        /// The default templates map, key is datatype, defulat templates information.
+        /// </summary>
+        private static IReadOnlyDictionary<DataType, DefaultTemplateInfo> _defaultTemplateMap = new Dictionary<DataType, DefaultTemplateInfo>
+        {
+            { DataType.Hl7v2, new DefaultTemplateInfo(DataType.Hl7v2, "microsofthealth/hl7v2templates:default", "Hl7v2DefaultTemplates.tar.gz") },
+            { DataType.Cda, new DefaultTemplateInfo(DataType.Cda, "microsofthealth/ccdatemplates:default", "CdaDefaultTemplates.tar.gz") },
```

Comment:
keep consist with cda? (ccda->cda)

Reply:
I will create a new pr to change cda to ccda, cda is used in this pr to keep consistent